main.go

```diff
@@ -19,9 +19,56 @@ limitations under the License.
 package main
 
 import (
-	"fmt"
+	"flag"
+	"os"
+
+	"sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm/controllers"
+
```

Comment:
Extra space

Reply:
grrrr i need to figure out what the HECK goland/goimports thinks it's doing